src/officehours_api/migrations/0015_profile_phone_number.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.0.7 on 2020-06-04 20:19
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('officehours_api', '0014_meeting_assignee'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='phone_number',
+            field=models.CharField(blank=True, default='', max_length=20),
+        ),
+        migrations.AlterField(
+            model_name='attendee',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='officehours_api.Profile'),
+        ),
+        migrations.AlterField(
+            model_name='meeting',
+            name='attendees',
+            field=models.ManyToManyField(through='officehours_api.Attendee', to='officehours_api.Profile'),
+        ),
+        migrations.AlterField(
+            model_name='attendee',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='meeting',
+            name='attendees',
+            field=models.ManyToManyField(through='officehours_api.Attendee', to=settings.AUTH_USER_MODEL),
+        ),
```

Comment:
Any idea why these `AlterField`s were generated or what they're doing? It looks like `attendee.user` and `meeting.attendees` have been changed to link to `Profile` instead of `User`? But I don't see any major changes in `models.py`.

Reply:
I changed them to link to Profile while I was playing around with things, then changed them back, that's why there's four of those. Only the first migration in this file actually does anything. Could I just delete the four AlterFields since they cancel each other out?